superbench/cli/_help.py

```diff
@@ -114,14 +114,14 @@
             --data-file outputs/results-summary.jsonl
             --rule-file rule.yaml
             --baseline-file baseline.json
-            --output-file-foramt excel
+            --output-file-format excel
```

Comment:
You can just create a separate PR for this bug fix

Reply:
ok